src/twisted/python/test/test_setup.py

```diff
@@ -127,12 +127,26 @@ def test_extrasRequiresDevDeps(self):
         L{_EXTRAS_REQUIRE}'s C{dev} extra contains setuptools requirements for
         the tools required for Twisted development.
         """
-        deps = _EXTRAS_REQUIRE["dev"]
-        self.assertIn("pyflakes >= 1.0.0", deps)
-        self.assertIn("twisted-dev-tools >= 0.0.2", deps)
-        self.assertIn("python-subunit", deps)
-        self.assertIn("sphinx >= 1.3.1", deps)
-        self.assertIn("twistedchecker >= 0.7.2", deps)
+        self.assertListEqual(
```

Comment:
wouldn't this be easier with:

```python
self.assertEqual(pathlib.Path(_setup.__file__).read_text() == """
    ...
""")
```

Reply:
I think that the purpose of this test is to check the high level dependenceis.

We should not care/test for `pillow` or `recommonmark`.
The important thing is to have `sphinx` and maybe `sphinx-rtd-theme` as top dependencies and the test should only check for that.

Reply:
I'm going to delete this test when I upgrade to `setup.cfg`